File: 250721/세 정수의 최솟값/minimum-of-three-numbers.java
// import
import java.util.Scanner; 

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int intA = sc.nextInt();
        int intB = sc.nextInt();
        int intC = sc.nextInt();

        // 최솟값 찾기
        int intMin;

        if (intA > intB) {
            if (intB > intC) {
                intMin = intC;
            } else {
                intMin = intB;
            }
        } else {
            if (intA > intC) {
                intMin = intC;
            } else {
                intMin = intA;
            }
        }

        // 출력
        System.out.println(intMin);
    }
}

File: 250721/세 정수의 최솟값/minimum-of-three-numbers.py
# 입력
intA, intB, intC = map(int, input().split())

# 변수 정의
intMin = 0

# 최소값 찾기
if intA > intB :
    if intB > intC :
        intMin = intC
    else :
        intMin = intB
else :
    if intA > intC :
        intMin = intC
    else :
        intMin = intA

# 출력
print(intMin)